integration/container/stop_windows_test.go

```diff
@@ -1,68 +0,0 @@
-package container // import "github.com/docker/docker/integration/container"
-
-import (
-	"context"
-	"strconv"
-	"testing"
-	"time"
-
-	containertypes "github.com/docker/docker/api/types/container"
-	"github.com/docker/docker/integration/internal/container"
-	"gotest.tools/v3/assert"
-	"gotest.tools/v3/poll"
-	"gotest.tools/v3/skip"
-)
-
-// TestStopContainerWithTimeout checks that ContainerStop with
-// a timeout works as documented, i.e. in case of negative timeout
-// waiting is not limited (issue #35311).
-func TestStopContainerWithTimeout(t *testing.T) {
-	skip.If(t, testEnv.OSType == "windows")
```

Comment:
Oh.. hmm.. interesting, so it was in a `windows` test-file, but it's skipped if the daemon is windows 😅